### PR TITLE
Fixes to SQL MIAA Compute + Storage page

### PR DESCRIPTION
**Pull request by candiceye**
- SyncSecondaryToCommit now a part of az sql mi-arc update
- Update field for SyncSecondaryToCommit now has a limit of 2
- Increment now works for SQL MIAA compute+storage page

https://user-images.githubusercontent.com/22386690/183537962-08399723-5a4b-443c-b3b3-c1b556018d16.mp4

### CODE REVIEW

**Review by alanrenmsft — APPROVED** [review]

File: extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts [hunk 1]
```diff
@@ -350,8 +350,7 @@ export class MiaaComputeAndStoragePage extends DashboardPage {
 			currentCPUSize = '';
 		}
 
-		this.coresRequestBox!.placeHolder = currentCPUSize;
-		this.coresRequestBox!.value = '';
+		this.coresRequestBox!.value = currentCPUSize;
```
Comment (alanrenmsft) [inline extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts:353]: please search and see whether you have other places in arc extension having such problem.

Comment (candiceye) [reply extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts:353]: I believe it behaves the same way in postgres. I'll submit a separate PR for that

File: extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts [hunk 2]
```diff
@@ -349,8 +350,9 @@ export class MiaaComputeAndStoragePage extends DashboardPage {
...
+		this.coresRequestBox!.placeHolder = '';
```
Comment (alanrenmsft) [inline extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts:354]: placeHolder is optional, you don't have to set it.

File: extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts [hunk 3]
```diff
@@ -323,7 +324,7 @@ export class MiaaComputeAndStoragePage extends DashboardPage {
 	}
 
 	private handleOnTextChanged(component: azdata.InputBoxComponent): boolean {
-		if ((!component.value)) {
+		if ((!component.placeHolder)) {
```
Comment (alanrenmsft) [inline extensions/arc/src/ui/dashboards/miaa/miaaComputeAndStoragePage.ts:327]: why are we changing this from value to placeholder?

**Pull request merged**